pkg/framework/host.go

```diff
@@ -61,17 +61,17 @@ Installation:
 `
 )
 
+type HostConfig struct {
```

Comment:
There is `Guest` and `Host` in this package. This is also not about the framework config but about the host config.